src/renderer/components/ft-toast/ft-toast.js

```diff
@@ -0,0 +1,42 @@
+import Vue from 'vue'
+import FtToastEvents from './ft-toast-events.js'
+
+export default Vue.extend({
+  name: 'FtToast',
+  data: function () {
+    return {
+      toasts: [],
+    }
+  },
+  mounted: function () {
+    FtToastEvents.$on('toast.open', this.open)
+  },
+  beforeDestroy: function () {
+    FtToastEvents.$off('toast.open', this.open)
+  },
+  methods: {
+    performAction: function (index) {
+      this.toasts[index].action()
+      this.remove(index)
+    },
+    close: function (toast) {
+      // Wait for fade-out to finish
+      setTimeout(this.remove, 300, this.toasts.length)
```

Comment:
Passing the length here will sometimes not work if toasts are coming in too fast.  Passing 0 like you did in the `open` function fixes this.